### ISSUE TITLE
select editable: defaultValue cannot be set to empty value

### ISSUE THREAD

**Issue by cadeyrn** [pimcore/pimcore#6459]
### Steps to reproduce / Code example:

Let's assume an areabrick with the following code:

```
{% set fooSelect = pimcore_select('foo', {
	store: [
                ['', '-- please select --'],
		['option 1', 'option 1'],
		['option 2', 'option 2']
	],
	defaultValue: '',
	reload: false
}) %}

{{ fooSelect|raw }}
```

### Actual behavior

fooSelect.getData() is `null` and the select field has no visible content.

### Expected behavior

fooSelect.getData() should be `""` and you can see "-- please select --" in the select field.

If I change the code to the following (not using the defaultValue parameter but setDataFromResource instead) it works as expected:

```
{% set fooSelect = pimcore_select('foo', {
	store: [
                ['', '-- please select --'],
		['option 1', 'option 1'],
		['option 2', 'option 2']
	],
	reload: false
}) %}
{% do fooSelect.isEmpty() ? fooSelect.setDataFromResource('') %}

{{ fooSelect|raw }}
```

Pimcore version: 6.6.4

**Comment by cadeyrn** [comment]
This bug still exists in Pimcoree 6.7.1.

**Comment by brusch** [comment]
Please be patient, the issue wasn't closed yet, so yes, why should it be fixed then in 6.7.1? 
If it is urgent for you we're happy to get, review and merge a PR 😉

**Comment by brusch** [comment]
Fixed by #7477
